### PR TITLE
[WIP] Add release pipline

### PR DESCRIPTION
**Pull request by oshercc**

### CODE REVIEW

File: tools/update_images_to_latest.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    # TODO add logsz
+    with open(args.deployment, "r") as f:
+        deployment = yaml.load(f)
+    for rep, rep_data in deployment.items():
+        project, repo = rep.split("/")
+        if project != "openshift":
+            continue
+        for image in rep_data["images"]:
+            image_full_name = IMAGE_FORMAT.format(image_name=image, tag=rep_data["revision"])
+            tag_image(image_full_name)
```
Comment (ronniel1) [inline tools/update_images_to_latest.py:31]: Maybe we should continue tagging even if on failed?

File: tools/update_images_to_latest.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+def tag_image(image):
+    subprocess.check_output("docker pull {}".format(image))
+    tagged_image = image.rsplit(":")[0] + args.tag
```
Comment (ronniel1) [inline tools/update_images_to_latest.py:35]: need to add ":" before the tag

**Comment by ronniel1** [comment]
@oshercc  had to use the tagging script so I fixed it and send a separate PR: https://github.com/eranco74/assisted-installer-deployment/pull/11

**Comment by oshercc** [comment]
@ronniel1 added a pr: https://github.com/eranco74/assisted-installer-deployment/pull/12

**Pull request closed without merging**